examples/js/loaders/GLTF2Loader.js

```diff
@@ -1668,10 +1693,12 @@ THREE.GLTF2Loader = ( function () {
 	GLTFParser.prototype.loadMeshes = function () {
 
 		var json = this.json;
+		var extensions = this.extensions;
 
 		return this._withDependencies( [
 
 			"accessors",
```

Comment:
Makes sense; done. We may eventually need to consider what should be returned, since the accessor cannot create a `THREE.BufferAttribute` in this case.